docs/docusaurus/docs/get_support.md

```diff
@@ -4,6 +4,12 @@ title: 'Get GX OSS support'
 description: Learn what resources are available to help you resolve issues with GX OSS.
 ---
 
+:::note Support during the holiday season
```

Comment:
Remove. We don't need it for OSS.

Reply:
@joshzzheng the note has been removed as requested.